demos/serverless/src/handlers.js

```diff
@@ -341,16 +342,98 @@ exports.end_capture = async (event, context) => {
     }).promise();
     return response(200, 'application/json', JSON.stringify({}));
   } else {
-    return response(500, 'application/json', JSON.stringify({ msg: "No pipeline to stop for this meeting" }))
+    return response(500, 'application/json', JSON.stringify({msg: "No pipeline to stop for this meeting"}))
+  }
+};
+
+
+exports.start_live_connector = async (event, context) => {
+  // Fetch the meeting by title
+  const meeting = await getMeeting(event.queryStringParameters.title);
+  meetingRegion = meeting.Meeting.MediaRegion;
+  const ivsChannelNamePrefix = IVS_CHANNEL_PREFIX?IVS_CHANNEL_PREFIX: "liveConnector"
```

Comment:
nit: `IVS_CHANNEL_PREFIX ? IVS_CHANNEL_PREFIX : "liveConnector"`

Reply:
discussed with my team. We don't need to use the provided prefix. Will just use `liveConnector `